src/Mod/PartDesign/Gui/TaskLoftParameters.cpp

```diff
@@ -105,13 +105,15 @@ TaskLoftParameters::TaskLoftParameters(ViewProviderLoft *LoftView, bool /*newObj
     if (profile) {
         Gui::Application::Instance->showViewProvider(profile);
 
+        // TODO: if it is a single vertex of a sketch, use that subshape's name

```

Comment:
I don't understand this comment. When I use a single-vertex sketch, the label is correctly the sketch. Like it is the case for all other cases in which sketches form a section. Therefore I don't see the need to use the vertex name, especially since you already assured that only single-vertex sketches can be used.

Reply:
I think you misunderstood. If the user selects a single vertex from a sketch (even if it has other elements), the code uses that instead of the whole sketch. It's not much relevant here (so I can remove it if I have to), but very useful for pipes, where we can choose one end point of a path as the first section and make a "horn" like shape.

Reply:
> If the user selects a single vertex from a sketch (even if it has other elements), 

Aha, cool feature, I was not yet aware of. Then it would indeed be useful to see the subname of the taken vertex and not the whole sketch as label. However, this is cosmetic. So leave the TODO in for now.